### PR TITLE
Updates/fixes to LVT USAFSIpost runmode for 557

### PR DESCRIPTION
**Pull request by emkemp**
### Description

This contains updates/fixes to running LVT in USAFSIpost runmode.

* Moved lvt.config.example to lvt/utils/afwa/usafsipost directory
* Updated lvt.config.example to add new line required by LVT to run
* Added logic in LVT to create output directory for GRIB files if directory doesn't exist already
* Added make_lvt_config_usafsipost.py to customize lvt.config file for valid date/time of USAFSI analysis
* Added submit_lvt_usafsipost_discover.py to create and submit batch job for running LVT in USAFSIpost runmode, on Discover.

### CODE REVIEW

File: lvt/runmodes/USAFSIpost/LVT_USAFSIpostMod.F90 [hunk 1]
```diff
@@ -396,8 +397,23 @@ subroutine output_grib2(this)
...
+      integer, external :: LVT_create_subdirs
+
+      ! Make sure output directory exists
+      inquire(file=trim(LVT_rc%output_dir), &
+           exist=found)
+      if (.not. found) then
+         rc = LVT_create_subdirs(len_trim(LVT_rc%output_dir), &
+              trim(LVT_rc%output_dir))
+         if (rc .ne. 0) then
+            write(LVT_logunit,*)'[WARN] Cannot create directory ', &
```
Comment (jvgeiger) [inline lvt/runmodes/USAFSIpost/LVT_USAFSIpostMod.F90:412]: If this routine cannot create the output directory, then should it issue an error message and call LVT_endrun?

File: lvt/runmodes/USAFSIpost/LVT_USAFSIpostMod.F90 [hunk 2]
```diff
@@ -528,6 +544,21 @@ subroutine interp_and_output_grib1(this, gridID)
...
+                 trim(LVT_rc%output_dir)
+         end if
+      end if
```
Comment (jvgeiger) [inline lvt/runmodes/USAFSIpost/LVT_USAFSIpostMod.F90:561]: Same comment as above.

If this routine cannot create the output directory, then should it issue an error message and call LVT_endrun?

File: lvt/runmodes/USAFSIpost/LVT_USAFSIpostMod.F90 (hunk 1 above)
Comment (emkemp) [reply lvt/runmodes/USAFSIpost/LVT_USAFSIpostMod.F90]: I concur.  I just pushed an update.

File: lvt/runmodes/USAFSIpost/LVT_USAFSIpostMod.F90 (hunk 2 above)
Comment (emkemp) [reply lvt/runmodes/USAFSIpost/LVT_USAFSIpostMod.F90:565]: I concur.  I just pushed an update.

**Pull request merged**